policy_definitions/update_management/policy_definition_qby_require_update_tags.json

```diff
@@ -0,0 +1,74 @@
+{
+    "name": "QBY-Require-Severity-Group-Tag",
+    "type": "Microsoft.Authorization/policyDefinitions",
+    "apiVersion": "2021-06-01",
+    "properties": {
+        "policyType": "Custom",
+        "mode": "All",
+        "displayName": "Deny Resources that are missing the severity group monthly tag.",
+        "description": "This policy ensures the tag for severiy group monthly is set and the general syntag is correct.",
+        "metadata": {
+            "version": "1.0.0",
+            "category": "Azure Update Manager"
+        },
+        "parameters": {
+            "osTypes": {
+                "type": "array",
+                "defaultValue": [
+                    "Windows"
+                ],
+                "metadata": {
+                    "description": "OS Types that get filtered for in the cinfiguration assignment."
+                }
+            },
+            "resourceTypes": {
+                "type": "array",
+                "defaultValue": [
+                    "microsoft.compute/virtualmachines"
+                ],
+                "metadata": {
+                    "description": "ResourceTypes that get filtered for in the cinfiguration assignment."
+                }
+            }
+        },
+        "policyRule": {
+            "if": {
+                "allOf": [
+                    {
+                        "field": "type",
+                        "in": "[parameters('resourceTypes')]"
+                    },
+                    {
+                        "anyOf": [
+                            {
+                                "field": "Microsoft.HybridCompute/machines/osName",
+                                "in": "[parameters('osTypes')]"
+                            },
+                            {
+                                "field": "Microsoft.Compute/virtualMachines/storageProfile.osDisk.osType",
+                                "in": "[parameters('osTypes')]"
+                            }
+                        ]
+                    },
+                    {
+                        "anyOf": [
+                            {
+                                "field": "[concat('tags[Severity Group Monthly]')]",
+                                "exists": false
+                            },
+                            {
+                                "not": {
+                                    "value": "[length(split(field(concat('tags[Severity Group Monthly]')),'-'))]",
```

Comment:
Fine for me. I can't think of a situation where a tag contains 6 segments and is not a valid severity group. But what would happen if someone enters false information in this tag?

Sadly regex matching doesn't seem to be supported and doing detailed validation without it would be very tedious.